build

```diff
@@ -3,10 +3,13 @@ set -euo pipefail
 
 COMMIT=$(git rev-parse HEAD)
 TAG=$(git describe --exact-match --abbrev=0 --tags "${COMMIT}" 2> /dev/null || true)
+BRANCH=$(git branch | grep \* | cut -d ' ' -f2 || true)
 OUTPUT_PATH=${OUTPUT_PATH:-"bin/kube-aws"}
+VERSION=""
 
 if [ -z "$TAG" ]; then
-	VERSION=$COMMIT
+        [[ -n "$BRANCH" ]] && VERSION="${BRANCH}/"
+	VERSION="${VERSION}${COMMIT:0:8}"
```

Comment:
Just curious but why 8 characters instead of 7, which I usually see from short commit IDs in GitHub?